src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -465,22 +468,42 @@ def test_acr_image_import(self, resource_group):
         Two source images each of which stays in a different source registries mentioned above.
         '''
         self.kwargs.update({
+            'resource_id': '/subscriptions/3fcdeaab-d87f-4a79-a70d-1c8fc67a2436/resourcegroups/resourcegroupdiffsub/providers/Microsoft.ContainerRegistry/registries/sourceregistrydiffsub',
+            'source_registry_rg': 'resourcegroupsamesub',
+            'source_loc': 'westus',
+            'source_registry_name': source_registry_name,
             'registry_name': registry_name,
-            'rg_loc': 'eastus',
             'sku': 'Standard',
-            'resource_id': '/subscriptions/a7ee80a4-3d5e-45c2-9378-36e8d98f4d13/resourceGroups/resourcegroupdiffsub/providers/Microsoft.ContainerRegistry/registries/sourceregistrydiffsub',
-            'source_image_diff_sub': 'builder:latest',
-            'source_image_same_sub': 'sourceregistrysamesub.azurecr.io/builder:latest',
-            'source_image_same_registry': '{}.azurecr.io/builder:latest'.format(registry_name),
-            'source_image_by_digest': 'sourceregistrysamesub.azurecr.io/builder@sha256:bc3842ba36fcc182317c07a8643daa4a8e4e7aed45958b1f7e2a2b30c2f5a64f',
-            'tag_diff_sub': 'repository_diff_sub:tag_diff_sub',
+            'rg_loc': 'eastus',
+            'source_image_diff_sub': 'microsoft:azure-cli',
+            'source_image_same_sub': '{}.azurecr.io/microsoft:azure-cli'.format(source_registry_name),
+            'source_image_same_registry': '{}.azurecr.io/microsoft:azure-cli'.format(registry_name),
+            'source_image_by_digest': '{}.azurecr.io/azure-cli@sha256:290dc12cab4d0a10032098d0f917c21cb9d8c8045d2e227d981851174944d118'.format(source_registry_name),
             'tag_same_sub': 'repository_same_sub:tag_same_sub',
             'tag_multitag1': 'repository_multi1:tag_multi1',
             'tag_multitag2': 'repository_multi2:tag_multi2',
             'tag_same_registry': 'repository_same_registry:tag_same_registry',
-            'tag_by_digest': 'repository_by_digest:tag_by_digest'
+            'tag_by_digest': 'repository_by_digest:tag_by_digest',
+            'source_image_public_registry_dockerhub': 'registry.hub.docker.com/library/hello-world',
```

Comment:
docker.io should work here

Reply:
Please refer to case 8 where I used "docker.io"

Reply:
Let's standardize on using docker.io for docker hub.  There's some special handling code needed for that hostname in the RP.